src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
+      const propertiesKeys = Object.keys(transforms[i].properties);
+
+      for (let j = 0; j < propertiesKeys.length; j++) {
+        state[propertiesKeys[j]] = transforms[i].properties[propertiesKeys[j]];
+      }
+    } else if (transforms[i].operation === 'clear') {
+      const stateKeys = Object.keys(state);
+
+      for (let k = 0; k < stateKeys.length; k++) {
+        delete state[stateKeys[k]];
+      }
+    } else if (transforms[i].operation === 'removeProperties') {
+      for (let m = 0; m < transforms[i].properties.length; m++) {
+        if (state.hasOwnProperty(transforms[i].properties[m])) {
+          delete state[transforms[i].properties[m]];
+        }
```

Comment:
```suggestion
          delete state[transforms[i].properties[m]];
```
There's no need to check if the 'state' has some property here. because if it hasn't, so 'delete' just will do nothing.